libs/payments/paypal/src/lib/error.ts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NVPError, NVPErrorResponse } from './types';
+
+// TODO: Create a dedicated error handling library in FXA-7656
+export class PayPalClientError extends Error {
+  public raw: string;
+  public data: NVPErrorResponse;
+  public errorCode: number | undefined;
+
+  constructor(raw: string, data: NVPErrorResponse, ...params: any) {
```

Comment:
Would be nice for us to have that shared library for error handling, then just have a translation here that parses PayPal error list similar to how this does and constructs our internal error lib.

Could actually properly handle multiple errors from PayPal rather than just picking the first, too.

I'm leaving this work for FXA-7656, however.